bundle/manifests/multicluster-engine.clusterserviceversion.yaml

```diff
@@ -3310,7 +3274,7 @@ spec:
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.namespace
-                image: quay.io/stolostron/backplane-operator:latest
+                image: quay.io/kurwang/backplane-operator:latest
```

Comment:
@kurwang We shouldn't be change the image reference related to the MCE CSV.

Reply:
Hi thanks Disaiah I missed this one, I removed it from the file